client/src/pages/Feed.js

```diff
@@ -108,16 +291,60 @@ const Feed = () => {
         handleFilterModal,
         handleQuit,
         handleLocation,
+        handleOnClose,
+        showFilters,
       }}
     >
       <FeedWrapper>
-        <FilterBox />
-        <Posts filteredPosts={fakePosts} />
-        <SvgIcon
-          src={creatPost}
-          onClick={handleCreatePost}
-          className="create-post"
-        />
+        <LayoutWrapper>
+          <SiderWrapper
+            breakpoint="md"
+            className="site-layout-background"
+            width={290}>
+            <div>
+              <MenuWrapper
+                defaultSelectedKeys={['ALL']}
+                onClick={handleChangeType}
+              >
+                {Object.keys(HELP_TYPE).map((item, index) =>
+                  <Menu.Item key={item}>
+                    {HELP_TYPE[item]}
+                  </Menu.Item>
+                )}
+              </MenuWrapper>
+              <FiltersWrapper>
+                <button
+                  onClick={handleShowFilters} >
+                  <span>
+                    <FiltersIcon />
+                  </span>
+                Filters
+              </button>
+                <FiltersList />
+              </FiltersWrapper>
+            </div>
+            <FiltersSidebar />
+          </SiderWrapper>
+          <ContentWrapper>
```

Comment:
I think the background-color you added is correct but not sure why it looks gray:

current
![Screen Shot 2020-05-11 at 5 07 52 PM](https://user-images.githubusercontent.com/42228386/81624045-083a8e00-93aa-11ea-9e60-5a53b84ced3a.png)

figma
![Screen Shot 2020-05-11 at 5 05 02 PM](https://user-images.githubusercontent.com/42228386/81624078-21433f00-93aa-11ea-850e-3ae45607ef95.png)

Reply:
In my screen:
<img width="300" alt="Captura de Pantalla 2020-05-11 a la(s) 11 22 17 p  m" src="https://user-images.githubusercontent.com/877561/81638317-65e8cd80-93de-11ea-973f-756372a69a66.png">

Reply:
On Figma it says it's `#F2F2F2` but I used a color picker to examine and it's #FBFBFB. I tried this value in the console and it looks correct now.

![Screen Shot 2020-05-11 at 11 00 25 PM](https://user-images.githubusercontent.com/42228386/81643790-35ebfb00-93db-11ea-9a7e-05956cb3b905.png)

Using #FBFBFB:
![Screen Shot 2020-05-11 at 11 01 45 PM](https://user-images.githubusercontent.com/42228386/81643859-66339980-93db-11ea-8c37-d932a653e36a.png)

Reply:
Changed to #FBFBD.